app/src/main/java/com/mapbox/navigation/examples/voice/PlayVoiceInstructionsActivity.kt

```diff
@@ -310,20 +305,20 @@ class PlayVoiceInstructionsActivity : AppCompatActivity() {
         speechApi = MapboxSpeechApi(
             this,
             getString(R.string.mapbox_access_token),
-            Locale.US.language
+            Locale.US.toLanguageTag()
```

Comment:
@Guardiola31337 like that?

Reply:
Yeah. In any case, @alexzatsepin could you confirm? 🙏

Reply:
Cc'ed you.

Reply:
> I doesn't work like that. The instructions are hardcoded in English. So if I change the locale to default (including the json) and set the system language to French, I'll get English instructions with French accent, which is super fun but not exactly what we want to hear.

@dzinad Probably you are right and instructions are not perfectly translated/pronounced. But keep in mind please, if you pass to json `en_US`(or just `en`), but the system locale will be English (United Kingdom) then instructions will be in American english, instead of British. For example, instead **junction** the client will be getting **exit** from Directions API, and this is may be a problem.